### DISCUSSION TITLE
Email

### DISCUSSION THREAD

**Discussion by jasonhollis** [autoSteve/acMqtt#52] [General]
You have now sent me down a rabbit hole on email LOL:

![CleanShot 2024-05-29 at 20 26 37@2x](https://github.com/autoSteve/acMqtt/assets/19743618/903803e4-7796-4959-b1bc-a8156a489cc2)

There is nothing wrong with my reputation score.  You have me on Red Sift trying to work out what I'm missing.  

I hope your well and enjoying the nice weather the last couple of days.  This 20 yr old alarm system has me pulling my hair out sir.

**Comment by jasonhollis** [comment]
Is it because I don't have DNSSEC enabled?

**Reply by autoSteve** [reply]
I am rather agressive with spam blocking. Maybe I've gone too far.

You're not a dirty rotten spammer are you? 😂

_"The RPBL Blocklist identifies spam-like sending practices and classifies emails based on specific criteria. These calculations are centered around sender behavior and content such as threatening attachments, spam trap hits, and authentication failures. An IP is listed when our system detects messages containing these indicators from that IP."_

**Reply by jasonhollis** [reply]
You have gone too far.

**Reply by autoSteve** [reply]
Zimbra reconfigured. Try again.

**Comment by jasonhollis** [comment]
Still a bit too tight sir:

mail.map59.com gave this error:
Decision Engine classified the mail item was rejected because of IP Block (from outbound normal IP pools) -> 554 5.7.1 Service unavailable; Client host [xx.xxx.xxx.xxx] blocked using bl.spamcop.net; Blocked

**Reply by autoSteve** [reply]
M$ outlook.com was listed on spamcop.

If I start getting spam I'm re-enabling it.

**Reply by jasonhollis** [reply]
??  I'm missing something.  My domains are hosted on 365 which is completely different than and outlook.com email address that is free.  The reason I'm paying stupid money is ... I will will explain in email but my stuff is not on outlook.com???

**Reply by autoSteve** [reply]
That doesn't mean outlook.com is not still being used.

```RCPT from mail-me3aus01on2104.outbound.protection.outlook.com[40.107.108.104]: <jason```

<img width="642" alt="image" src="https://github.com/autoSteve/acMqtt/assets/37229860/93a88d62-c41c-42ef-b9d5-e44449731029">

**Comment by jasonhollis** [comment]
Thinking about making a change myself.  I'm paying Microsoft like $150/mo for 5 email accounts and two Office licenses.  There are obviously many office alternatives and the other call it $100 for email is over the top.

**Comment by jasonhollis** [comment]
I'm guessing that you are not using Zimbra Cloud?  

![CleanShot 2024-05-30 at 14 07 33@2x](https://github.com/autoSteve/acMqtt/assets/19743618/568247b1-1b7c-4bd2-8617-c86ced959d01)

**Comment by jasonhollis** [comment]
Sorry I just refreshed the page.  Looking at MX Toolbox now which is what I wanted to find yesterday.  I think I understand the issue now.

**Comment by jasonhollis** [comment]
I thought this was failing because I didn't have a DMARC policy.  What is not cool is there is no distinction between outlook.com and office365.  Short of you getting into the weeds and configuring things very specifically.  The level of security that MS applies to office 365 by default is very good and very different than a free account on outlook.com.  What a dumb thing.

**Comment by jasonhollis** [comment]
My security is specific to my subdomain.   That is very different than *.outlook.com.

**Comment by jasonhollis** [comment]
just do a lookup on MX toolbox for my domains.
